pkg/controller/azvolumeattachment.go

```diff
@@ -762,6 +786,13 @@ func (r *reconcileAzVolumeAttachment) CleanUpAzVolumeAttachment(ctx context.Cont
 	}
 
 	volRequirement, _ := labels.NewRequirement(VolumeNameLabel, selection.Equals, []string{azVolume.Spec.UnderlyingVolume})
```

Comment:
Same as above. Should it volRequirement, err?

Reply:
Fixed